src/reference/asciidoc/file.adoc

```diff
@@ -105,6 +106,19 @@ to, say, network glitches.
 </bean>
 ----
 
+*Message Headers*
+
+Starting with _version 5.0_ the `FileReadingMessageSource`, in addition to the `payload` as a polled `File`, populates these headers to the outbound `Message`:
+
+- `FileHeaders.FILENAME` - the `File.getName()` of the file to send.
+Can be used for subsequent rename or copy logic;
+- `FileHeaders.ORIGINAL_FILE` - the `File` object itself.
+Typically this header is populated automatically by Framework components, like <<file-splitter>> or <<file-transforming>>, when we lose the original `File` object.
+But for consistency and convenience with any other custom use-cases this header can be useful to get access to the original file;
+- `FileHeaders.RELATIVE_PATH` - a new header introduced to represent the part of file path without the root directory for scan.
```

Comment:
`s/without the root directory for scan/relative to the root directory of the scan/`